packages/config/config/devices/0x031e/lzw31-sn.json

```
@@ -363,174 +361,33 @@
 		},
 		{
 			"#": "15",
-			"label": "LED Indicator Brightness When Off",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 10,
-			"defaultValue": 1,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Off",
-					"value": 0
-				},
-				{
-					"label": "10%",
-					"value": 1
-				},
-				{
-					"label": "20%",
-					"value": 2
-				},
-				{
-					"label": "30%",
-					"value": 3
-				},
-				{
-					"label": "40%",
-					"value": 4
-				},
-				{
-					"label": "50%",
-					"value": 5
-				},
-				{
-					"label": "60%",
-					"value": 6
-				},
-				{
-					"label": "70%",
-					"value": 7
-				},
-				{
-					"label": "80%",
-					"value": 8
-				},
-				{
-					"label": "90%",
-					"value": 9
-				},
-				{
-					"label": "100%",
-					"value": 10
-				}
-			]
+			"$import": "#paramInformation/14",
+			"label": "LED Indicator: Brightness When Off"
 		},
 		{
-			"#": "16[0xff]",
-			"label": "LED Effect Color",
-			"description": "Uses a scaled hue value (realHue / 360 * 255).",
-			"valueSize": 4,
-			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 0,
-			"options": [
-				{
-					"label": "Red",
-					"value": 0
-				},
-				{
-					"label": "Orange",
-					"value": 21
-				},
-				{
-					"label": "Yellow",
-					"value": 42
-				},
-				{
-					"label": "Green",
-					"value": 85
-				},
-				{
-					"label": "Cyan",
-					"value": 127
-				},
-				{
-					"label": "Blue",
-					"value": 170
-				},
-				{
-					"label": "Violet",
-					"value": 212
-				},
-				{
-					"label": "Pink",
-					"value": 234
-				},
-				{
-					"$if": "firmwareVersion >= 1.47",
-					"label": "White",
-					"value": 255
-				}
-			]
+			"#": "16[0x000000ff]",
```

We usually leave out leading zero bytes in the bitmasks
```suggestion
			"#": "16[0xff]",
```